docs/index.rst

```diff
@@ -8,73 +8,41 @@
 ROCm installation for Linux
 ****************************************************************
 
-If you're new to ROCm, we recommend using the :ref:`rocm-install-quick`. If you want more
-complete instructions, refer to the following sections.
+This section describes the ROCm for Linux installation options.
 
-.. grid:: 2
-    :gutter: 1
-
-    .. grid-item-card:: Quick start
-        :link: tutorial/quick-start
-        :link-type: doc
+.. grid:: 1
+    :gutter: 3
 
-        ROCm quick start installation guide.
-
-    .. grid-item-card:: Prerequisites
-        :link: how-to/prerequisites
-        :link-type: doc
+    .. grid-item-card:: Install
 
-        Steps required before installation.
+       * :doc:`Quick start <tutorial/quick-start>` - recommended if you're new to ROCm
+       * :doc:`Detailed <tutorial/detailed-install>` - if you require complete installation information
```

Comment:
I would suggest the hierarchy of related topics to be displayed as a sub-list here.

Reply:
I prototyped it and it looked too cluttered.  We ended up removing it.